src/routes/Routes.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react'
+import { Route, useHistory } from 'react-router-dom'
+import Question from '../views/Question'
+import Home from '../views/Home'
+import Login from '../views/Login'
+import Signup from '../views/Signup'
+import AskQuestion from '../views/AskQuestion'
+import About from '../views/About'
+import NotFound from '../views/404'
+import Switch from 'react-bootstrap/esm/Switch'
+import ReactGA from 'react-ga'
+
+export default () => {
+  const history = useHistory()
+  ReactGA.initialize('UA-204918741-1')
```

Comment:
I think you should do this outside of the component to prevent it from happening multiple times